deploy-vercel.sh

```
@@ -0,0 +1,30 @@
+# deploy.sh — Unified deploy script for Vercel
+# Usable in CI or locally
+#
+# Requirements:
+#   - VERCEL_TOKEN must be set (env var or GitHub Secret)
+#   - Project must be linked with Vercel (see below)
+
+set -euo pipefail
+
+# Load local env vars from .env if present
+if [ -f .env ]; then
+  export $(grep -v '^#' .env | xargs)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)